app/lib/shared/versions.dart

```diff
@@ -22,7 +22,7 @@ final RegExp runtimeVersionPattern = RegExp(r'^\d{4}\.\d{2}\.\d{2}$');
 /// when the version switch happens.
 const acceptedRuntimeVersions = <String>[
   // The current [runtimeVersion].
-  '2023.08.17',
+  '2023.08.18',
```

Comment:
Is it safe to fallback when we've added a new property?

Reply:
Yes, it is. (Note: this needs to be rebased and updated since we've merged the deploy PR).

Reply:
updated